apps/services/handlers.py

```diff
@@ -277,3 +278,38 @@ async def read_request_details(university_id: int, user_request_id: int, user=De
         "data": response,
         "message": "Got request details"
     }
+
+
+@services_router.post("/{user_request_id}/student-accommodation/{university_id}/",
+                      name="count_student_accommodation",
+                      response_model=JSENDOutSchema[CountHostelAccommodationIn],
+                      tags=["Student dashboard"])
+async def count_student_accommodation(university_id: int, user_request_id: int,  hostel: CountHostelAccommodationIn,
```

Comment:
1) count_hostel_accommodation_cost is better
2) delete param user_request_id
3) hostel is the worst name of the variable. Please rename it to data for example